app/core/Tracker2/DriftTracker.js

```diff
@@ -111,11 +112,14 @@ export default class DriftTracker extends BaseTracker {
   }
 
   get isChatEnabled () {
-    return !this.onPlayPage && !this.store.getters['me/isStudent']
+    return !this.onPlayPage && !this.store.getters['me/isStudent'] && !this.store.getters['me/isHomePlayer']  // && !this.disableAllTracking
   }
 
   updateDriftConfiguration () {
     const chatEnabled = this.isChatEnabled
+    if (chatEnabled && !this.driftApi && this.isInitialized !== false) {
+      return this.initializeDrift()
+    }
 
     window.drift.config({
```

Comment:
On student accounts we are now getting the error on this line: `Cannot read property 'config' of undefined`. This is happening because we no longer load Drift at all for them. Thus we need an early return for that case just above this line:

`if (!chatEnabled && !window.drift) { return; }`

Reply:
or simpler may work as well: `if (!window.drift) { return; }`